packages/@react-native-windows/cli/src/config/dependencyConfig.ts

```diff
@@ -271,47 +271,69 @@ export function dependencyConfigWindows(
       const projectContents = configUtils.readProjectFile(projectFile);
 
-      const projectName = configUtils.getProjectName(
+      const projectType = configUtils.getProjectType(
         projectFile,
         projectContents,
       );
 
-      const projectGuid = configUtils.getProjectGuid(projectContents);
+      if (projectType === 'dynamiclibrary' || projectType === 'winmdobj') {
+        const projectLang = configUtils.getProjectLanguage(projectFile);
 
-      const projectNamespace = configUtils.getProjectNamespace(projectContents);
+        const projectName = configUtils.getProjectName(
+          projectFile,
+          projectContents,
+        );
 
-      const directDependency = true;
+        const projectGuid = configUtils.getProjectGuid(projectContents);
 
-      const cppHeaders: string[] = [];
-      const cppPackageProviders: string[] = [];
-      const csNamespaces: string[] = [];
-      const csPackageProviders: string[] = [];
+        const projectNamespace = configUtils.getProjectNamespace(
+          projectContents,
+        );
 
-      if (projectNamespace !== null) {
-        const cppNamespace = projectNamespace.replace(/\./g, '::');
-        const csNamespace = projectNamespace.replace(/::/g, '.');
+        const directDependency = true;
 
-        cppHeaders.push(`winrt/${csNamespace}.h`);
-        cppPackageProviders.push(`${cppNamespace}::ReactPackageProvider`);
-        csNamespaces.push(`${csNamespace}`);
-        csPackageProviders.push(`${csNamespace}.ReactPackageProvider`);
-      }
+        const cppHeaders: string[] = [];
+        const cppPackageProviders: string[] = [];
+        const csNamespaces: string[] = [];
+        const csPackageProviders: string[] = [];
 
-      result.projects.push({
-        projectFile: path.relative(sourceDir, projectFile),
-        projectName,
-        projectLang,
-        projectGuid,
-        directDependency,
-        cppHeaders,
-        cppPackageProviders,
-        csNamespaces,
-        csPackageProviders,
-      });
+        if (projectNamespace !== null) {
+          const cppNamespace = projectNamespace.replace(/\./g, '::');
+          const csNamespace = projectNamespace.replace(/::/g, '.');
+
+          cppHeaders.push(`winrt/${csNamespace}.h`);
+          cppPackageProviders.push(`${cppNamespace}::ReactPackageProvider`);
+          csNamespaces.push(`${csNamespace}`);
+          csPackageProviders.push(`${csNamespace}.ReactPackageProvider`);
+        }
+
+        result.projects.push({
+          projectFile: path.relative(sourceDir, projectFile),
+          projectName,
+          projectLang,
+          projectGuid,
+          directDependency,
+          cppHeaders,
+          cppPackageProviders,
+          csNamespaces,
+          csPackageProviders,
+        });
+      } else {
+        const projectPath = path.relative(sourceDir, projectFile);
+        result.projects.push({
+          projectFile: `Error: ${projectPath} is type '${projectType}'`,
```

Comment:
I'm not following this error - we couldn't figure out the root namespace for the project, but this error seems to be about project type instead?